pkg/networkplugin-syncer/handlers/ovn/handler.go

```diff
@@ -65,10 +61,6 @@ func (ovn *SyncHandler) Init() error {
 		return err
 	}
 
-	if err := ovn.detectOvnKubernetesImplementation(); err != nil {
-		return err
-	}
-
```

Comment:
Cool. So the implementation is going to work with any version of ovn-kubernetes?

Reply:
Yup :-D :-) that's the plan, so far it seems that works for the old CI and new (OCP 4.8) I need to try an older one (OCP 4.5, but not a blocker), and I think that technically should work.

Reply:
Correct :crossed_fingers: